functorch/functorch/_src/eager_transforms.py

```diff
@@ -785,11 +785,20 @@ def jvp(func: Callable, primals: Any, tangents: Any, *, strict: bool = False, ha
          >>> assert torch.allclose(output, x + y)
 
     """
+
+    return _jvp_with_argnums(func, primals, tangents, argnums=None, strict=strict, has_aux=has_aux)
+
+
+# only to be used with jacrev to allow for all passed variables to be wrapped correctly, even if they don't participate
```

Comment:
jacrev -> jacfwd

Reply:
Also, it's not true that this is only used in jacfwd. `_jvp_with_argnums` is used for jacfwd and for jvp

Reply:
Fixed to match the new `_vjp_with_argnums` comment (more verbose). Also double checked that all vjp -> jvp and jacrev -> jacfwd (and a couple other language things for tangents, return values)